src/main/views/template.njk

```diff
@@ -13,10 +13,10 @@
         homepageUrl: "#",
         serviceName: "Find a court or tribunal listing",
         serviceUrl: "#",
-        navigationClasses: 'nav-right',
+        navigationClasses: 'govuk-header__navigation--end nav-right',
```

Comment:
ooo good find! youve left in `nav-right` though, may as well just extend the gov one to have the -4vh? ill approve either way, let you decide if you want 2 classes or not

Reply:
amended css to overwrite gov.uk css to use one css class